scene/2d/cpu_particles_2d.cpp

```diff
@@ -1232,6 +1257,9 @@ void CPUParticles2D::_notification(int p_what) {
 					_interpolation_data.global_xform_curr = get_global_transform();
 				}
 			}
+#ifdef TOOLS_ENABLED
+			queue_redraw();
```

Comment:
Not needed, moving the node will move the gizmo together with it.
Unless it serves some purpose?

Reply:
As far as I can tell this is needed as the rendered gizmo does not follow CPU particles when local_coords is disabled.